pkg/template/servicebroker/test-scripts/shared.sh

```diff
@@ -6,5 +6,5 @@ bindingUUID=${bindingUUID-dde0226b-ff95-4f9d-af51-2e9ec06b1f02}
 
 endpoint=${endpoint-https://localhost:8443/brokers/template.openshift.io}
 curlargs=${curlargs--k}
-namespace=${namespace-demo}
+namespace=${namespace-myproject}
```

Comment:
sorry, again, the default of `myproject` is what is in play when bringing up via `oc cluster up` ... where is `demo` coming from ?